bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,187 @@
+| device-local-ssos3-udc            | Number:ElectricPotential| R          | DC link voltage as measured by SSO-3     |
+| device-local-ssos3-faultcode      | String                  | R          | See section 4.1.3.1                      |
+| device-local-ssos3-ipv            | Number:ElectricCurrent  | R          | Measured on PV string side               |
+| device-local-ssos3-upv            | Number:ElectricPotential| R          | Measured on PV string side               |
+| device-local-ssos3-id             | String                  | R          | Unique identifier of SSO-3               |
+| device-local-eso-faultcode        | String                  | R          | See section 4.1.3.1                      |
+| device-local-eso-id               | String                  | R          | Unique identifier                        |
+| device-local-eso-ibat             | Number:ElectricCurrent  | R          | Measured on battery side                 |
+| device-local-eso-ubat             | Number:ElectricPotential| R          | Measured on battery side                 |
+| device-local-eso-relaystatus      | String                  | R          | 0 = relay closed (i.e running power), 1 = relay open/disconnected, 2 = precharge|
+| device-local-eso-soc              | Number:Dimensionless    | R          | State of Charge for ESO                  |
+| device-local-eso-temp             | Number:Temperature      | R          | Measured inside ESO                      |
+| device-local-eso-wpbatcons        | Number:Energy           | R          | Total energy produced by ESO, i.e total energy charged|
+| device-local-eso-wbatprod         | Number:Energy           | R          | Total energy produced by ESO, i.e total energy discharged|
+| device-local-eso-udc              | Number:Power            | R          | DC link voltage as measured by ESO       |
+| device-local-eso-ts               | String                  | R          | Time stamp when message was published    |
+| device-local-esm-soh              | Number:Dimensionless    | R          |                                          |
+| device-local-esm-soc              | Number:Dimensionless    | R          |                                          |
+| device-local-esm-rated-capacity   | Number:Energy           | R          | Rated capacity of battery                |
+| device-local-esm-id               | String                  | R          | Unique identifier of battery. If available, this will be the unique id that the battery reports. |
+| device-local-esm-rated-power      | Number:Power            | R          | Rated power of battery                   |
+| device-local-esm-status           | String                  | R          | Dependent on battery manufacturer        |
+| device-local-esm-ts               | String                  | R          | Time stamp when message was published    |
+  
```

Comment:
```suggestion
```

Beside the remove of this double empty line, I would suggest to add a ## examnple section. With the large amount of channels it's probably to extensive to create a fulll example, creating a bridge/thing with some channels will be of good help to users.